apps/db/src/main/resources/db/migration/V21__inntektsmelding_add_im_nytt_format.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE inntektsmelding
+    ADD COLUMN inntektsmelding JSONB,
+    ADD COLUMN erProsessert    BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Ville kanskje tenkt "prosessert" er bedre kolonnenavn? Men ingen biggie.

Reply:
For boolean så synes jeg en `er`-prefix er bedre, men det slo meg at den kolonnen heller kan har en LocalDateTime, så ser vi når prosessering er ferdig.